subprojects/logging/src/main/java/org/gradle/internal/logging/LoggingManagerInternal.java

```diff
@@ -50,4 +50,9 @@
     LoggingManagerInternal captureStandardError(LogLevel level);
 
     LoggingManagerInternal setLevelInternal(LogLevel logLevel);
+
+    /**
+     * Sets the worker count to display in the UI.
+     */
+    LoggingManagerInternal setMaxWorkerCount(int maxWorkerCount);
```

Comment:
`visibleWorkerCount()`? Note that we're not setting the height, but the ideal number of items in-progress (depending on actual console size), and it's not tied to workers (that is, we don't assign 1 worker to 1 line, so we don't need to mention "workers" anywhere.